src/v2/Apps/Show/Components/ShowContextCard.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowContextCard_show } from "v2/__generated__/ShowContextCard_show.graphql"
+import { FairTimingFragmentContainer as FairTiming } from "v2/Apps/Fair/Components/FairHeader/FairTiming"
+import { FairCardFragmentContainer as FairCard } from "v2/Components/FairCard"
+import { StyledLink } from "v2/Components/Links/StyledLink"
+
+interface Props {
+  show: ShowContextCard_show
+}
+
+export const ShowContextCard: React.FC<Props> = ({ show }) => {
+  const { isFairBooth, fair } = show
+
```

Comment:
Rather than ternaries at the return, we generally return early in our components since its easier to see whats going on:
```tsx
if (!fair) {
  return null
}
```